src/backend/jitsi_magnify/core/api.py

```diff
@@ -32,42 +37,69 @@ def exception_handler(exc, context):
 
     return drf_exception_handler(exc, context)
 
-class CustomTokenSerializer(TokenObtainPairSerializer):
-    """Api viewset to generate a custom token"""
 
-    @classmethod
-    def get_token(cls, user, room):
-        """Method to get the token created based on a user and a room which they wish to access"""
-        token = super().get_token(user)
+def create_payload(user, room):
+    """Create the payload so that it contains each information jitsi requires"""
+    expiration_minutes = settings.JWT_CONFIGURATION.get("token_expiration_minutes")
+    expiration_hours = settings.JWT_CONFIGURATION.get("token_expiration_hours")
+    expiration_days = settings.JWT_CONFIGURATION.get("token_expiration_days")
```

Comment:
Actually, you can just use seconds to ease configuration. It does not prevent us from setting expiration to 259200 seconds if we want 3 days :man_shrugging: